source-code/ide-extension/src/commands/extractMessage.ts

```diff
@@ -14,17 +16,26 @@ export const extractMessageCommand = {
 
 		// guards
 		if (!ideExtension) {
-			return vscode.window.showWarningMessage(
+			return msg(
 				"There is no `ideExtension` object in the inlang.config.json configured.",
+				"warn",
+				"notification",
 			)
 		}
 		if (ideExtension.extractMessageOptions === undefined) {
-			return vscode.window.showWarningMessage(
-				"The `extractMessageReplacementOptions` are not defined in the inlang.config.json but required to extract a message.",
-			)
```

Comment:
I would keep this in and not set a default. The default will break, leading to frustrated developers. A concise error message on why the IDE Extension doesn't work seems much better.

Reply:
@samuelstroschein So people have to paste in a huge config when using the inlang extension when there's no plugin (like it is now?) and no sdk usage?

Reply:
> So people have to paste in a huge config when using the inlang extension 

Yes. 

Don't you think the DX is better when the ide extension nudges a user to install a plugin that works rather than pretending to work but not work? The latter leads to "this extension is buggy and crap".